toxcore/group.c

```diff
@@ -880,13 +882,23 @@ static void rejoin_frozen_friend(Group_Chats *g_c, int friendcon_id)
     }
 }
 
+static int g_handle_any_status(void *object, int friendcon_id, uint8_t status, void *userdata)
```

Comment:
I think this is already in master, no? Perhaps rebase so it doesn't appear in this PR?

Reply:
No, I closed that PR. This change is only needed for saving.